core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -93,6 +99,9 @@
     private Function<? super EventLoopGroup,
             ? extends AddressResolverGroup<? extends InetSocketAddress>> addressResolverGroupFactory =
             DEFAULT_ADDRESS_RESOLVER_GROUP_FACTORY;
+    private int initialHttp2ConnectionRcvWindow = DEFAULT_WINDOW_SIZE;
```

Comment:
I don't think we need to do so (until they remove them from their public API, which is not likely at the moment.)